src/mixins/stripe.js

```diff
@@ -64,6 +64,14 @@ var StripeMixin = {
       label: error
     });
   },
+  reCaptcha: function(callback) {
+    grecaptcha.render('g-recaptcha', {
```

Comment:
So thinking about putting the content of this function inside of some check to make sure `grecaptcha` is on the window, this way we can lazy load the captcha script while the user interacts with the form itself, likely being ready by the time they are done.

Reply:
would that be difficult?